classes/system/nagios/server/single.yml

```diff
@@ -0,0 +1,9 @@
+applications:
+  - nagios
+classes:
+- service.nagios.server.single
+parameters:
+  _param: {}
```

Comment:
on this level add param to support heka_alarm collection, look at grafana for example how to set this https://github.com/tcpcloud/salt-formula-grafana/ the remote_data with the engine.